Integration/Scripted REST Api/DomainSeperation/README.md

```diff
@@ -0,0 +1,147 @@
+---
+# PR Description:
+
+This pull request adds a Scripted REST API for ServiceNow that creates Incident records in the domain and company of the authenticated user. The API was developed as part of a Hacktoberfest 2025 contribution and is intended for educational and demonstration purposes. The `create` method ensures incidents are scoped to the user's domain, supporting multi-tenancy and domain separation best practices.
+---
+
+# Pull Request Checklist

```

Comment:
Please do not include PR template information in the readme file as it is not relevant to the code snippet itself